subsys/net/lib/mqtt_sn/Kconfig

```diff
@@ -0,0 +1,59 @@
+# MQTT-SN Library for Zephyr
+
+# Copyright (c) 2022 René Beckmann
+# SPDX-License-Identifier: Apache-2.0
+
+config MQTT_SN_LIB
+	bool "MQTT-SN Library Support"
```

Comment:
That's a good idea, we could keep the library experimental for a few releases so that we have more freedom finetuning the API if needed. @BeckmaR Here's an example on how to set a library as experimental: https://github.com/zephyrproject-rtos/zephyr/blob/f483b1bc4cfe2381624fac6337cc74d3ce2bc850/subsys/net/lib/tftp/Kconfig#L6 (note the `select EXPERIMENTAL` is also needed).

Reply:
Done that, makes sense. Thanks for the input.